worker/tern_worker.js

```diff
@@ -177,7 +246,7 @@ handler.analyze = function(value, ast, callback, minimalAnalysis) {
     };
     addTernFile(this.path, value);
 
-    architectResolver.onReady(function() {
+    architectResolver && architectResolver.onReady(function() {
```

Comment:
I do not know much about architectResolver tern-plugin.
This code block resides in *handler.analyze* method, which is an event to be called for handling the code changes to be analyzed. As the existing code calls this every time, so the change there should call it every time as the *architectResolver* object is present